Index: src/test/java/org/skillbrain/AppTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package org.skillbrain;\r\n\r\nimport junit.framework.Test;\r\nimport junit.framework.TestCase;\r\nimport junit.framework.TestSuite;\r\n\r\n/**\r\n * Unit test for simple App.\r\n */\r\npublic class AppTest \r\n    extends TestCase\r\n{\r\n    /**\r\n     * Create the test case\r\n     *\r\n     * @param testName name of the test case\r\n     */\r\n    public AppTest( String testName )\r\n    {\r\n        super( testName );\r\n    }\r\n\r\n    /**\r\n     * @return the suite of tests being tested\r\n     */\r\n    public static Test suite()\r\n    {\r\n        return new TestSuite( AppTest.class );\r\n    }\r\n\r\n    /**\r\n     * Rigourous Test :-)\r\n     */\r\n    public void testApp()\r\n    {\r\n        assertTrue( true );\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/org/skillbrain/AppTest.java b/src/test/java/org/skillbrain/AppTest.java
--- a/src/test/java/org/skillbrain/AppTest.java	(revision 6721e5e599d997fa724e9d0d4e27c6df36456d89)
+++ b/src/test/java/org/skillbrain/AppTest.java	(date 1677613999705)
@@ -1,31 +1,28 @@
 package org.skillbrain;
 
-import junit.framework.Test;
-import junit.framework.TestCase;
-import junit.framework.TestSuite;
 
 /**
  * Unit test for simple App.
  */
-public class AppTest 
-    extends TestCase
+public class AppTest
+        // extends TestCase
 {
     /**
      * Create the test case
      *
      * @param testName name of the test case
      */
-    public AppTest( String testName )
+    // public AppTest( String testName )
     {
-        super( testName );
+        // super( testName );
     }
 
     /**
      * @return the suite of tests being tested
      */
-    public static Test suite()
+    //  public static Test suite()
     {
-        return new TestSuite( AppTest.class );
+        // return new TestSuite( AppTest.class );
     }
 
     /**
@@ -33,6 +30,6 @@
      */
     public void testApp()
     {
-        assertTrue( true );
+        // assertTrue( true );
     }
 }
Index: src/main/java/Utilitis/Utils.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/Utilitis/Utils.java b/src/main/java/Utilitis/Utils.java
new file mode 100644
--- /dev/null	(date 1677610617259)
+++ b/src/main/java/Utilitis/Utils.java	(date 1677610617259)
@@ -0,0 +1,56 @@
+package Utilitis;
+
+import org.apache.commons.io.FileUtils;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+import java.io.File;
+import java.io.IOException;
+import java.time.Duration;
+import java.time.LocalDateTime;
+
+
+public  class Utils {
+    // pentru print screen
+    public static  WebDriver getChromeDriver() {
+        ChromeOptions options=new ChromeOptions();
+         options.addArguments("start-maximized");
+        options.addArguments("headless");
+        // options.setHeadless(true);
+        System.setProperty("Chromedriver.chrome.driver"," .idea/drivers/chromedriver.exe");
+        WebDriver driver=new ChromeDriver();
+         driver.manage().window().maximize();
+         driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        return driver;
+    }
+    //  public void takeScreenshot(WebDriver driver) throws IOException {
+    public static void takeScreenshot(WebDriver driver) throws IOException {
+        try {
+            String destinationFile = null;
+            TakesScreenshot screenshot =  (TakesScreenshot) driver;
+            File source = screenshot.getScreenshotAs(OutputType.FILE);
+            destinationFile = System.getProperty("user.dir");
+
+            destinationFile= System.getProperty("user.dir")+"C:\\Users\\stelu\\IdeaProjects\\Helin-QAC-g1-17-11-22\\imagine.png"
+                    + "/Logs/printScreen-" + LocalDateTime.now() + ".png";
+            FileUtils.copyFile(source, new File(destinationFile));
+
+
+        }catch (IOException e) {
+          e.printStackTrace();
+
+    }
+
+        }
+}
+
+
+
+
+
+
+
+
Index: src/test/java/org/skillbrain/AT101.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/org/skillbrain/AT101.java b/src/test/java/org/skillbrain/AT101.java
new file mode 100644
--- /dev/null	(date 1677860882946)
+++ b/src/test/java/org/skillbrain/AT101.java	(date 1677860882946)
@@ -0,0 +1,16 @@
+package org.skillbrain;
+import Utilitis.Utils;
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.WebDriver;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import PageObjects.DemoqaPage;
+
+import java.io.IOException;
+
+public class AT101 extends UtiL{
+
+
+
+}
Index: .idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Default ignored files\r\n/shelf/\r\n.idea/\r\n/workspace.xml\r\n.idea/**/workspace.xml\r\n.idea/**/tasks.xml\r\n.idea/**/usage.statistics.xml\r\n.idea/**/dictionaries\r\n.idea/**/shelf\r\nout/\r\n.idea/artifacts\r\n.idea/compiler.xml\r\n.idea/jarRepositories.xml\r\n.idea/modules.xml\r\n.idea/*.iml\r\n.idea/modules\r\n*.iml\r\n*.ipr\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
--- a/.idea/.gitignore	(revision 6721e5e599d997fa724e9d0d4e27c6df36456d89)
+++ b/.idea/.gitignore	(date 1677614587407)
@@ -1,18 +1,1 @@
 # Default ignored files
-/shelf/
-.idea/
-/workspace.xml
-.idea/**/workspace.xml
-.idea/**/tasks.xml
-.idea/**/usage.statistics.xml
-.idea/**/dictionaries
-.idea/**/shelf
-out/
-.idea/artifacts
-.idea/compiler.xml
-.idea/jarRepositories.xml
-.idea/modules.xml
-.idea/*.iml
-.idea/modules
-*.iml
-*.ipr
Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\r\ntarget/\r\n.DS_Store\r\nsrc/main/\r\nLogs/\r\n .idea/drivers/\r\n .idea/libraries/\r\n .idea/uiDesigner.xml\r\n  src/main/java/Alerts.java\r\n  src/main/java/PageObjects/EmagHomePage.java\r\n  src/main/java/Utilitis/\r\n  src/main/java/org/skillbrain/Notes.java\r\n  src/main/java/org/skillbrain/Tema 2/\r\n  src/main/java/org/skillbrain/Tema 3/\r\n  src/main/java/org/skillbrain/Tema4/Selenium2punct1.java\r\n  .idea/drivers/\r\n  .idea/libraries/\r\n   .idea/uiDesigner.xml\r\n\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 6721e5e599d997fa724e9d0d4e27c6df36456d89)
+++ b/.gitignore	(date 1677613295096)
@@ -1,20 +1,7 @@
 
 target/
 .DS_Store
-src/main/
+
 Logs/
- .idea/drivers/
- .idea/libraries/
- .idea/uiDesigner.xml
-  src/main/java/Alerts.java
-  src/main/java/PageObjects/EmagHomePage.java
-  src/main/java/Utilitis/
-  src/main/java/org/skillbrain/Notes.java
-  src/main/java/org/skillbrain/Tema 2/
-  src/main/java/org/skillbrain/Tema 3/
-  src/main/java/org/skillbrain/Tema4/Selenium2punct1.java
-  .idea/drivers/
-  .idea/libraries/
-   .idea/uiDesigner.xml
 
 
